tasks/fix-cat2.yml

```diff
@@ -40,54 +36,75 @@
       - dod_logon_banner
 
 - name: "MEDIUM | RHEL-07-010060 | PATCH | The Red Hat Enterprise Linux operating system must enable a user session lock until that user re-establishes access using established identification and authentication procedures."
-  command: "true"
-  changed_when: no
+  blockinfile:
+      create: yes
+      dest: /etc/dconf/db/local.d/00-screensaver
+      block: |
+        [org/gnome/desktop/screensaver]
```

Comment:
yamllint doesn't catch block-strings, but I'd tend to prefer 4 indents here instead of 2; minor nit, though.

Reply:
You are correct, this is a minor nit.